runners/samza/src/main/java/org/apache/beam/runners/samza/translation/GroupByKeyTranslator.java

```diff
@@ -65,6 +65,12 @@ public void translate(
     doTranslate(transform, node, ctx);
   }
 
+  static class KeyedWorkItems<K, InputT>
+  implements TransformTranslator<PTransform<
+      PCollection<KV<K, InputT>>, PCollection<KeyedWorkItem<K, InputT>>>> {
+    doTra
```

Comment:
is "doTra" a typo here?

Reply:
This was not supposed to be part of the commit at all and was in my scratch space.